src/encoded/upgrade/hic_quality_metric.py

```diff
@@ -0,0 +1,62 @@
+from snovault import upgrade_step
+
+
+@upgrade_step('hic_quality_metric', '1', '2')
+def hic_quality_metric_1_2(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-6183
+    if 'alignable_normal_and_chimeric_paired' in value:
+        value['2_alignments'] = value['alignable_normal_and_chimeric_paired']
+        value.pop('alignable_normal_and_chimeric_paired')
```

Comment:
Way too much code here to change it all, but you can actually do this in one line for future reference:
```suggestion
        value['2_alignments'] = value.pop('alignable_normal_and_chimeric_paired')
```